psi4/src/psi4/libmints/mintshelper.cc

```diff
@@ -454,16 +448,11 @@ void MintsHelper::one_body_ao_computer(std::vector<std::shared_ptr<OneBodyAOInt>
         const auto *ints_buff = ints[rank]->buffers()[0];
 
         size_t index = 0;
-        if (symm) {
-            for (size_t mu = index_mu; mu < (index_mu + num_mu); ++mu) {
-                for (size_t nu = index_nu; nu < (index_nu + num_nu); ++nu) {
-                    outp[nu][mu] = outp[mu][nu] = ints_buff[index++];
-                }
-            }
-        } else {
-            for (size_t mu = index_mu; mu < (index_mu + num_mu); ++mu) {
-                for (size_t nu = index_nu; nu < (index_nu + num_nu); ++nu) {
-                    outp[mu][nu] = ints_buff[index++];
+        for (size_t mu = index_mu; mu < (index_mu + num_mu); ++mu) {
+            for (size_t nu = index_nu; nu < (index_nu + num_nu); ++nu) {
+                outp[mu][nu] = ints_buff[index++];
+                if (symm) {
+                    outp[nu][mu] = outp[mu][nu];
```

Comment:
Why change this? Although there is less code duplication, the unrolled `if` is traditionally faster.

Reply:
I'll change this back, I didn't know the unrolled if would be faster. This will help with my other projects as well :)